Mathlib/Data/List/BigOperators/Basic.lean

```diff
@@ -13,7 +13,7 @@ import Mathlib.Data.List.Forall2
 
 This file provides basic results about `List.prod`, `List.sum`, which calculate the product and sum
 of elements of a list and `List.alternating_prod`, `List.alternating_sum`, their alternating
-counterparts. These are defined in [`Data.List.Defs`](./defs).
+counterparts. These are defined in [`Data.List.Defs`](Mathlib/Data/List/Defs).
```

Comment:
That should be enough, right?
```suggestion
counterparts. These are defined in [`Data.List.Defs`](./Defs).
```

Reply:
Ah, so the relative paths should actually work? I though the HTML generator will always insert the four `../` and thus assume a path relative from the top level, starting with `Mathlib` :thinking: